### ISSUE TITLE
Creating new section Entertainment

### ISSUE THREAD

**Issue by RedHeadphone** [public-apis/public-apis#2320]
In [Personality section](https://github.com/public-apis/public-apis#personality) there are many api which doesn't seem fit like NaMoMemes, Yo Momma Jokes, Imgflip and Biriyani As A Service. So I thought we can make a section of Entertainment. If it seems right I would like to separate them.

**Comment by shangkaul** [comment]
> In [Personality section](https://github.com/public-apis/public-apis#personality) there are many api which doesn't seem fit like NaMoMemes, Yo Momma Jokes, Imgflip and Biriyani As A Service. So I thought we can make a section of Entertainment. If it seems right I would like to separate them.

Yep, there are many APIs related to movies-tv series like star wars and IMDB API in the Video section, these also would be better suited in the Entertainment section.

**Comment by pawelborkar** [comment]
@matheusfelipeog  @marekdano What you guys think on this?

**Comment by marekdano** [comment]
> @matheusfelipeog @marekdano What you guys think on this?

If there are more APIs that can be in a new section then we should do that. I think it's better to have more sections for the specific APIs than a general one.

**Comment by RedHeadphone** [comment]
Biriyani As A Service would be better if it's in [food drink section](https://github.com/public-apis/public-apis#food--drink) right
